cvat/apps/quality_control/quality_reports.py

```diff
@@ -265,6 +265,14 @@ def precision(self) -> float:
     def recall(self) -> float:
         return self.valid_count / (self.gt_count or 1)
 
```

Comment:
What about accuracy? Why only precision and recall?

Reply:
The overall accuracy value is not the average of per-class accuracies. It's going to be smaller than each of them, actually, because it has the same denominator, but the numerator is smaller. (Conceptually, it's harder to be a multiclass classifier than a binary classifier for each class.)

Theoretically, you could average the per-class accuracies and call that a "macro average accuracy", but I'm not sure that's a useful metric. 🤷‍♂️

Reply:
What do you mean by overall accuracy? Micro accuracy?

> Theoretically, you could average the per-class accuracies and call that a "macro average accuracy", but I'm not sure that's a useful metric. 🤷‍♂️

Yes, that's what I'm suggesting to add.